UNRELEASED.md

```diff
@@ -4,6 +4,9 @@
 
 **Added:**
 
+- react-native-bpk-component-text:
```

Comment:
Can you please move this to a `**Fixed:**` block?

Reply:
Yes ! will do